services/bettersocial/post/blockAnonymousPost.js

```diff
@@ -54,8 +53,8 @@ const BetterSocialBlockAnonymousPost = async (token, selfUserId, postId, source,
       await UserFollowUserFunction.userBlock(
         UserFollowUser,
         UserFollowUserHistory,
-        selfUserId,
         authorAnonymousUserId,
+        selfUserId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of arguments passed to `UserFollowUserFunction.userBlock` has been changed. This could potentially lead to incorrect behavior if the function expects arguments in a specific order. Please ensure that the function `userBlock` can handle this change in argument order.



<!-- This is an auto-generated comment by OSS CodeRabbit -->